### PR TITLE
TV Show Search

### PR DESCRIPTION
**Pull request by sejalxz**
## Related Issue
  - Added TV Show Search Project

Closes: #501 

### Describe the changes you've made
* I added the necessary JavaScript, HTML and CSS files for my TV Show Search project.
* Updated script.js

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
Tv_Show_Search

### CODE REVIEW

**Comment by sejalxz** [comment]
@991rajat @urvashi-code1255 I have resolved the conflicts. Please look into it. Thanks :)

File: script.js [hunk 1]
```diff
@@ -344,12 +344,17 @@ let projectData = [
...
-  ,
   {
     projectName: "Typewriter Effect",
     projectImage: "assets/GIFs/typewriter.gif",
     projectUrl: "public/typewriter.html",
   },
+  {
+    projectName: "TV Show Search",
+    projectImage: "assets/GIFs/TV_Show_Search.gif",
+    projectUrl: "public/TV_Show_Search.html",
```
Comment (991rajat) [inline script.js:355]: Not Properly Linked. Typo Error

**Review by 991rajat — CHANGES_REQUESTED** [review]
Good Work
One Suggestion Could you it centered in mobile View

**Comment by 991rajat** [comment]
@sejalxz Similar PR is opened by you. I'm closing this.

**Pull request closed without merging**